packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -1917,16 +1917,14 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin,
   @protected
   bool hitTestChildren(BoxHitTestResult result, { required Offset position }) {
     final InlineSpan? textSpan = _textPainter.text;
-    if (textSpan != null) {
-      final Offset effectivePosition = position - _paintOffset;
-      final TextPosition textPosition = _textPainter.getPositionForOffset(effectivePosition);
-      final Object? span = textSpan.getSpanForPosition(textPosition);
-      if (span is HitTestTarget) {
+    final Offset effectivePosition = position - _paintOffset;
+    switch (textSpan?.getSpanForPosition(_textPainter.getPositionForOffset(effectivePosition))) {
+      case final HitTestTarget span:
         result.add(HitTestEntry(span));
         return true;
-      }
+      case _:
```

Comment:
is there a different between this and `default`?

Reply:
I think here they are the same.

Reply:
in general i recommend leaning away from `_` where possible, fwiw. It's harder to understand at a glance. (Not sure if `default` works here though.)

Reply:
> Not sure if default works here though.

`default` does work here I believe.  

> It's harder to understand

How so? Wildcards can be used with other patterns (e.g. `case _ when x > 0 : `) and in other patterns (e.g. `[_, a, b, _]`), so they're more powerful than `default` and in switch expressions I think `default` is not allowed (e.g.  `switch(num) { 42 => true, default => false };` is not valid). So using `_` syntax in switch statements to catch all is more consistent with switch expressions I feel?